services/web/client/source/class/qxapp/data/Store.js

```diff
@@ -50,7 +50,8 @@ qx.Class.define("qxapp.data.Store", {
     getRole: function() {
       // Temporary HACK
       const userEmail = qxapp.auth.Data.getInstance().getEmail();
-      if (userEmail.includes("itis.swiss") || userEmail.includes("oetiker.ch")) {
+      const inHouse = ["itis.swiss", "speag.swiss", "oetiker.ch"];
```

Comment:
mmhhh I would not hard-code this. Can u pass it as a configuration variable upon compilation?

Reply:
This is just a temporary hack until the user resource gets in, I wouldn't overdo it

Reply:
please add a TODO or FIXME